cmd/edit/addon/cmd.go

```diff
@@ -129,69 +126,29 @@ func run(cmd *cobra.Command, argv []string) {
 
 		var val string
 		var hasVal bool
+		input := &interactive.Input{
```

Comment:
using an input outside of interactive mode only to get the options seems strange to me

Reply:
I'm declaring the object here in order to use it in the validation later. I can make a change where it just an empty object and put in the values inside the interactive block.